src/main.ts

```diff
@@ -127,7 +127,7 @@ async function getAIResponse(prompt: string): Promise<Array<{
     const response = await openai.chat.completions.create({
       ...queryConfig,
       // return JSON if the model supports it:
-      ...(OPENAI_API_MODEL === "gpt-4-1106-preview"
+      ...(OPENAI_API_MODEL === "gpt-4-turbo-preview" || OPENAI_API_MODEL === "gpt-4-turbo" || OPENAI_API_MODEL === "gpt-3.5-turbo" || OPENAI_API_MODEL === "gpt-4-0125-preview" || OPENAI_API_MODEL === "gpt-4-1106-preview" || OPENAI_API_MODEL === "gpt-3.5-turbo-0125" || OPENAI_API_MODEL === "gpt-3.5-turbo-1106"
```

Comment:
Oh this was a good catch! Where did you find the list? 🤔 Looks like most if not all the chat models are listed?

Reply:
It's a combination of information listed [here](https://platform.openai.com/docs/guides/text-generation/json-mode), [here](https://platform.openai.com/docs/models/gpt-3-5-turbo), and [here](https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo).

It is basically all of the current-version turbo models. The standard models (e.g., `gpt-4`) still do not support JSON mode.